packages/govtnz-ds-website/src/commons/constants.js

```diff
@@ -0,0 +1,10 @@
+const GA_MEASUREMENT_ID = 'UA-109491125-3';
+
+const getGaEventTarget = id => {
+    return `gtag_${id.replace(new RegExp('-', 'g'), '_')}`;
+};
+
+module.exports = {
+    GA_MEASUREMENT_ID,
+    GA_EVENT_TARGET: getGaEventTarget(GA_MEASUREMENT_ID),
+};
```

Comment:
Could you use ES modules for this? (not CJS), thanks!

Reply:
I've changed this to JSON so it's not CJS but can still be imported by Node and Webpack.